easybuild/easyconfigs/m/MXNet/MXNet-1.9.1-foss-2022a.eb

```diff
@@ -0,0 +1,67 @@
+name = 'MXNet'
+version = '1.9.1'
+
+homepage = 'https://mxnet.io/'
+description = """Flexible and Efficient Library for Deep Learning"""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+toolchainopts = {'cstd': 'c++14', 'opt': True, 'pic': True, 'openmp': True, 'extra_cflags': '-lflexiblas'}
+
+source_urls = [
+    'https://github.com/apache/mxnet/archive',
+    'https://github.com/oneapi-src/oneDNN/archive/',
+]
+sources = [
+    'v%(version)s.tar.gz',  # MXNet
+    'https://github.com/dmlc/ps-lite/archive/34fd45cae457d59850fdcb2066467778d0673f21.tar.gz',  # ps-lite
+    'https://github.com/apache/tvm/archive/efdac9439506d1de5eec91ecc795982c78e41909.tar.gz',  # tvm
+    'https://github.com/dmlc/dmlc-core/archive/5df8305fe699d3b503d10c60a231ab0223142407.tar.gz',  # dmlc-core
+    'https://github.com/dmlc/dlpack/archive/3efc489b55385936531a06ff83425b719387ec63.tar.gz',  # dlpack
+    'https://github.com/google/googletest/archive/eb9225ce361affe561592e0912320b9db84985d0.tar.gz',  # googletest
+    'https://github.com/kpu/intgemm/archive/8f28282c3bd854922da638024d2659be52e892e9.tar.gz',  # intgemm
+    'https://github.com/NVlabs/cub/archive/0158fa19f28619886232defd412433974af89611.tar.gz',  # cub
+    'https://github.com/onnx/onnx-tensorrt/archive/2eb74d933f89e1590fdbfc64971a36e5f72df720.tar.gz',  # onnx-tensorrt
+    # oneDNN:
+    {
+        'download_filename': '5818c40f07bdb6307f9bc64e929836fe036da644.tar.gz',
+        'filename': 'mkldnn.tar.gz'
```

Comment:
also no good, will collide names. This needs a unqiue name to indicate what version it is.  Is it also really important that it's this specific commit? oneDNN have proper releases and tarballs like normal.

Reply:
fixed...and yeah, the thing is that this specific commit is being pulled by the original repo, so I suppose it's best to keep it that way...